client/plasmacli/include.go

```diff
@@ -19,21 +20,35 @@ func init() {
 }
```

Comment:
Need to add `includeCmd.Flags().String(flagAccount, "", "description")`. Also I think we should use a flag with the value "address" rather than "account" since a user might try to input the name of their account rather than a hex address